vue-app/src/components/TimeLeft.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <span :class="valueClass || 'value'">
+    {{ values[0] }}
+    <span :class="unitClass || 'unit'">
+      {{ units[0] }}
+    </span>
+    <span v-if="units[1].length > 0">
+      {{ units[1].length > 0 && values[1] }}
```

Comment:
```suggestion
      {{ values[1] }}
```

Reply:
I believe this check is redundant - the v-if statement on the line above checks for this.

Reply:
Good catch, thanks